### DISCUSSION TITLE
Flow not show in webpage went update Akvorado to v1.11

### DISCUSSION THREAD

**Discussion by AndyZ1608** [akvorado/akvorado#1359] [Q&A]
Hi Vincent,
Glad to see you again.
Our team is using Akvorado run standalone in 3 VM.1 for Kafka v3.5 , 1 for ClickHouse v22.8 and 1 for Akvorado. We have plan to update Akvorado to v1.11 and we do this last weekend. Update successfully and Akvorado change to v1.11 but the flow and exporter become 0.
I think the problem about version of ClickHouse and Kafka. I checked with some API but it not reponse any errors.
Thank you so much.
AndyZ

P/s: We used to update from v1.9 to v1.10 before and it's still okay.

**Comment by vincentbernat** [comment]
It is difficult to guess with so little information. Look at the troubleshooting guide and explain what works and what not.

**Reply by AndyZ1608** [reply]
![image](https://github.com/user-attachments/assets/c88575b1-3f36-4946-9672-8062d2207c07)
[
![image (1)](https://github.com/user-attachments/assets/0c15de95-f050-4005-81d4-d5e7982f5fbd)
](url)
 Hi Vincent, i just try to update again and i think root cause from Kafka. Our version is 3.5.0 and service still okay,but when update to v1.11 the webpage do not show any flow and exxporters.

**Reply by AndyZ1608** [reply]
![image (2)](https://github.com/user-attachments/assets/ff3d7de7-5cf1-4cbb-be01-7d5447fcac1b)
Update I have try to update to version 1.10.0 and we have 3 exporters but in webpage only show 1.
